packages/crayons-core/src/components/accordion-body/accordion-body.scss

```diff
@@ -4,13 +4,15 @@ $accordion-border-radius: var(--fw-accordion-border-radius, 8px);
   @prop --fw-accordion-body-font-size: Accordion body font size
   @prop --fw-accordion-body-line-height: Accordion body line height
   @prop --fw-accordion-body-background-color: Accordion body background color
-  @prop --fw-accordion-body-padding: Accordion body padding
+  @prop --fw-accordion-body-horizontal-padding: Accordion body left - right padding
+  @prop --fw-accordion-body-vertical-padding: Accordion body top - bottom padding
 */
 .accordion-body {
   font-size: var(--fw-accordion-body-font-size, 14px);
   line-height: var(--fw-accordion-body-line-height, 18px);
   background-color: var(--fw-accordion-body-background-color, #f5f7f9);
-  padding: var(--fw-accordion-body-padding, 20px);
+  padding-inline: var(--fw-accordion-body-horizontal-padding, 20px);
```

Comment:
Hi, this will be a breaking change since we are removing the already existing css variable. Can you pls keep have the --fw-accordion-body-padding as well to avoid thebreaking change ?.

Reply:
Fixed.